src/fillTank.test.js

```diff
@@ -1,11 +1,188 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('should fill tank fully with no amout', () => {
```

Comment:
The test description 'should fill tank fully with no amout' contains a typo. It should be 'amount' instead of 'amout'.

Reply:
done